internal/util/util.go

```diff
@@ -8,3 +12,27 @@ func ToPtrSlice[T any](slice []T) []*T {
 	}
 	return result
 }
+
+func ToPtr[T any](x T) *T {
+	return &x
+}
+
+// SliceToCommaSeparated converts a string slice to a comma-separated string.
+// If quoteItems is true, each item in the slice will be quoted.
+func SliceToCommaSeparated(slice []string, quoteItems bool) string {
+	if len(slice) == 0 {
+		return ""
+	}
+	if !quoteItems {
+		return strings.Join(slice, ",")
+	}
```

Comment:
You could use `strings.Join()` for both cases, and it would be even shorter, e.g.
```
func SliceToCommaSeparated(slice []string, quoteItems bool) string {
        if quoteItems {
                quoted := []string{}
                for _, s := range slice {
                        quoted = append(quoted, "'"+s+"'")
                }
                return strings.Join(quoted, ",")
        }

        return strings.Join(slice, ",")
}
```
This will work correctly (for quoting and adding commas) on slices having 0, 1, or >1 elements.

Reply:
addressed